docs/docsite/rst/user_guide/playbooks_best_practices.rst

```diff
@@ -28,15 +28,15 @@ Directory Layout
 
 The top level of the directory would contain files and directories like so::
 
-    production                # inventory file for production servers
-    staging                   # inventory file for staging environment
+    production.ini            # inventory ini file for production servers
+    staging.ini               # inventory ini file for staging environment
```

Comment:
there are 2 reasons these do not have extensions:
- .ini files are ignored by default when parsing directories (this is because inventory scripts normally have .ini configs)
- we kept the files ambiguous on purpose so users could have inventories in ini/yaml or toml formats, we are not establishing as specific one in this doc

Reply:
I see, makes sense. It would help to add this info into this doc.